### ISSUE TITLE
"No applicable devices found" When exporting to ipa using xcodebuild because of a missing UIDeviceFamily in Instabug plist

### ISSUE THREAD

**Issue by alaeddineG** [Instabug/Instabug-iOS#124]
Our app is currently using Instabug (6.2.1) 
What happen is when we try to export the app with the following command in out build server
`xcodebuild -exportArchive -exportPath  the_app.ipa -archivePath the_app_archive.xcarchive -exportOptionsPlist export_options_adhoc.plist` 

We face this error : 
`Error Domain=IDEDistributionErrorDomain Code=14 "No applicable devices found." UserInfo={NSLocalizedDescription=No applicable devices found.}`
After some research I found out that someone have faced the same issue with Google cast sdk wish is package the same way as Instabug in cocoapod (providing only the .framework ) : https://code.google.com/p/google-cast-sdk/issues/detail?id=954

I had to make sure which pod is causing the problem and I create and empty project with all the pods we are using in the app, tried to disable all pods first time to make sure the export work fine then tried to build/archive/export ipa with only one pod enable at the time. All of the attempt succeeded with the export only when it came to Instabug the export failed with the error mentioned before. 

I had to use the same workaround to make the export working : 
`plutil -replace UIDeviceFamily -xml "<array><integer>1</integer><integer>2</integer></array>" Info.plist`
on the level of `/Pods/Instabug/Instabug.framework/Instabug.bundle/Info.plist` and `/Pods/Instabug/Instabug.framework/Info.plist `

We will appreciate if you added the UIDeviceFamily on the plist file contained in the packages Instabug.framework in a future release. ( so far we have to live with the workaround as  a Build phase in xcode )

Thank you.

**Comment by Figo0o** [comment]
Hello @alaeddineG, Thanks for reporting that out. 

Can you please update to the latest Instabug Version. This issue was fixed. (starting from **v6.3.1**)

Thanks!

**Comment by alaeddineG** [comment]
Oh ! my bad ! Thank you ! 
I've updated to 7 lately (2 weeks ago) but I had some freezing issues. I hope those have been fixed.

**Comment by Figo0o** [comment]
No Worriers! 

Yes :( .. We are so sorry for that. We are releasing V7.0.3 very soon. Let me notify you then.

**Comment by alaeddineG** [comment]
It's was 7.0.0 (and 6.4.1) I had problem with actually (in development fortunately). I didn't had time to test 7.0.2. I'am doing it right now. 
Is the freezing issue when attaching a file with the report when the app is not debugging is fixed?. I've repported this issue to the support 1 week ago. I've been told that this have been fixed in 7.0.1 !  So I don't have to wait until 7.0.3. If yes when 7.0.3 will be released then?

Chokran ! :)
